detection_rules/rule.py

```diff
@@ -698,6 +698,9 @@ class EQLRuleData(QueryRuleData):
     """EQL rules are a special case of query rules."""
     type: Literal["eql"]
     language: Literal["eql"]
+    timestamp_field: Optional[str]
+    event_category_override: Optional[str]
+    tiebreaker_field: Optional[str]
```

Comment:
Do these fields default to anything? My default settings have timestamp_field and event_category_override set. 

<img width="324" alt="Screenshot 2023-07-12 at 8 16 39 AM" src="https://github.com/elastic/detection-rules/assets/1636709/e149cf5e-aecf-4521-9511-f121a25c60d8">

There's also another field `size`. Are we planning on setting that?

Reply:
This is only for EQL rules, not timelines. `size` is not available as an option for EQL rules. There are no default values set it appears when doing EQL rules, only timelines.

<img width="1056" alt="Screenshot 2023-07-12 at 12 57 55 PM" src="https://github.com/elastic/detection-rules/assets/99630311/c94fb071-889f-4a77-81bf-3b79572a1db9">